clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -1589,7 +1603,7 @@ protected void checkIngestionProgress(Store store) throws InterruptedException {
       }
       return;
     }
-    idleCounter = 0;
+    idleCounter.set(0);
```

Comment:
nit: maybe call `resetIdleCounter()`

Reply:
Removed this function instead since there is only one caller. It doesn't seem necessary to switch function context for this update.